files/messages/welcome.txt

```diff
@@ -1,13 +1,11 @@
 Hello everyone, and thank you for signing up for our upcoming instructor training session. In order to prepare:
 
-1.  Please read:
+1.  Please read these two short papers, which provide a brief overview of some key evidence-based results in teaching:
 
-    * "Success in Introductory Programming: What Works?" (https://swcarpentry.github.io/instructor-training/files/papers/porter-what-works-2013.pdf)
-    * "Top 10 Myths About Teaching Computer Science" (https://swcarpentry.github.io/instructor-training/files/papers/guzdial-10-myths-2015.pdf)
     * "The Science of Learning" (https://swcarpentry.github.io/instructor-training/files/papers/science-of-learning-2015.pdf)
-    * "Software Carpentry: Lessons Learned" (http://f1000research.com/articles/3-62/v2)
+    * "Success in Introductory Programming: What Works?" (https://swcarpentry.github.io/instructor-training/files/papers/porter-what-works-2013.pdf)
 
-2.  Please go to the Software Carpentry lessons page (http://software-carpentry.org/lessons/) and the Data Carpentry lessons page (http://www.datacarpentry.org/lessons/) and have a look at what we currently teach.  You are *not* expected to go through all of these in detail, but please familiarize yourself with the major topics and lesson format.
+2.  Please go to the Software Carpentry lessons page (http://software-carpentry.org/lessons/) and the Data Carpentry lessons page (http://www.datacarpentry.org/lessons/), have a look at what we currently teach, and then choose one episode from the list at the bottom of this message and read through it carefully---we will ask you to do some exercises based on your choice on the first day of the course.
```

Comment:
Minor changes (but for some reason I can't add changes to this doc - weird permissions issue?).

1. Change "carefully---" to "carefully."

2. Put an emphasis on "one" in "one episode" so there's no potential confusion about having to read through all. 

3. Change last sentence to "You will be using your selected episode for several in-class exercises, so be sure you are comfortable with the content."